src/status_im2/navigation/core.cljs

```diff
@@ -328,6 +332,13 @@
    (navigation/pop (name @state/root-comp-id))
    (navigate view-id)))
 
+(fx/defn toggle-local-pairing-experimental-mode
+  {:events [:toggle-local-pairing-experimental-mode]}
```

Comment:
why this event in navigation? and why reloaded involved? do we really want to use async storage ?